modules/npm/src/main/typescript/main.d.ts

```diff
@@ -1,8 +1,12 @@
-declare class PlotPoint {
+export declare class PlotPoint {
   constructor(instant: Date, airmass: number, altitude: number);
```

Comment:
If `PlotPoint` is not a `js.Object` then I don't think it has a JS-accessible ctor?